src/components/Calculator.js

```diff
@@ -17,20 +17,23 @@ const Calculator = () => {
   };
 
   return (
-    <div className="container">
-      <div className="input">
-        <input
-          type="text"
-          placeholder="0"
-          id="display"
-          required
-          className="field"
-          value={calculatorState.next || calculatorState.total || ''}
-          readOnly
-        />
+    <>
+      <h1 className="title">Calculator is Powerful Machine. Use Wisely!</h1>
+      <div className="container">
+        <div className="input">
+          <input
+            type="text"
+            placeholder="0"
+            id="display"
+            required
+            className="field"
+            value={calculatorState.next || calculatorState.total || ''}
+            readOnly
+          />
+        </div>
+        <Btn onClick={handleButtonClick} />
       </div>
-      <Btn onClick={handleButtonClick} />
-    </div>
+    </>
```

Comment:
- [ ] @the-faizmohammad Great work adding stlye to your calculator page, however there are sight differences to the design given, check the comparison below for better understanding:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/the-faizmohammad/MathsMagicians/assets/61408860/a84845a7-5f1d-4aff-86f5-b03b76697298)  | ![image](https://github.com/the-faizmohammad/MathsMagicians/assets/61408860/803da5c9-ec17-4cfe-99a8-8cc77d306384)